WcaOnRails/app/mailers/competitions_mailer.rb

```diff
@@ -112,8 +112,8 @@ def submit_report_nag(competition)
     @competition = competition
     mail(
       to: competition.delegates.pluck(:email),
-      cc: ["board@worldcubeassociation.org"] + delegates_to_senior_delegates_email(competition.delegates),
-      reply_to: "board@worldcubeassociation.org",
+      cc: ["quality@worldcubeassociation.org"] + delegates_to_senior_delegates_email(competition.delegates),
+      reply_to: delegates_to_senior_delegates_email(competition.delegates),
```

Comment:
Just checking: is this `reply_to` intentional? Are you sure we don't want `quality@` to get these replies?

Reply:
The Seniors are the ones expected to follow up on those emails, so I think the `reply_to` field should point to them.